protomap/proto.go

```diff
@@ -725,6 +787,17 @@ func makeWrapper(msg protoreflect.Message, fd protoreflect.FieldDescriptor, val
 	}
 }
 
+// isWrapper returns true if the field fd of the message msg is a ywrapper protobuf type.
+func isWrapper(msg protoreflect.Message, fd protoreflect.FieldDescriptor) bool {
+	newV := msg.NewField(fd)
+	switch newV.Message().Interface().(type) {
+	case *wpb.StringValue, *wpb.UintValue, *wpb.BytesValue, *wpb.BoolValue:
```

Comment:
Are Decimal64Value, IntValue missing?

Another idea to potentially make this less error-prone is
```go
newV.Message().FullName() matches "ywrapper.\w+Value"
```
But given ywrapper changes so infrequently I'm also fine with the current.

Reply:
They are missing -- will add them.

I thought about doing something similar, but I was wondering whether that actually is less detectable that this code needs to change if ywrapper did change in some backwards incompatible manner. I'll prefer to leave it as it as the moment (with the type additions), but we can re-examine.

Reply:
Done.